.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_CLIENT_ID=YOUR_CLIENT_ID
```

Comment:
Could we consolidate down to one .env file that is shared by both the frontend and backend? The CLIENT_ID should always be the same for the frontend and backend.